packages/frontend/src/features/metricsCatalog/hooks/useMetricChartAnalytics.tsx

```diff
@@ -0,0 +1,43 @@
+import {
+    type ApiCatalogAnalyticsResults,
+    type ApiError,
+} from '@lightdash/common';
+import { useQuery } from '@tanstack/react-query';
+import { lightdashApi } from '../../../api';
+
+const getMetricChartAnalytics = async ({
+    projectUuid,
+    table,
+    field,
+}: {
+    projectUuid: string;
+    table: string;
+    field: string;
+}) => {
+    return lightdashApi<ApiCatalogAnalyticsResults>({
+        url: `/projects/${projectUuid}/dataCatalog/${table}/analytics/${field}`,
+        method: 'GET',
+        body: undefined,
+    });
+};
+
+export const useMetricChartAnalytics = ({
```

Comment:
we have something similar in `dataCatalog` feature but it's a mutation - and a query will just work here bc it's only available when there's a table & a field